### ISSUE TITLE
segfault

### ISSUE THREAD

**Issue by videlec** [Normaliz/PyNormaliz#23]
The following consistently produces a segfault
```
>>> from PyNormaliz import Cone, NmzCone, NmzResult
>>> V = [[-1, -1, 1], [-1, 0, 1], [-1, 1, 1], [0, -1, 1], [0, 0, 1], [0, 1, 1], [1, -1, 1], [1, 0, 1], [1, 1, 1]]
>>> cube1 = Cone(vertices=V)
>>> cube2 = NmzCone(vertices=V)
>>> NmzResult(cube2, "SupportHyperplanes") == [[-1, 0, 1], [0, -1, 1], [0, 1, 1], [1, 0, 1]]
True
>>> NmzResult(cube2, "Volume") == [[8, 1]]
True
>>> cube1.Triangulation()
```

**Comment by jplab** [comment]
Inside sage, I get:

sage: from PyNormaliz import Cone, NmzCone, NmzResult
sage: V = [[-1, -1, 1], [-1, 0, 1], [-1, 1, 1], [0, -1, 1], [0, 0, 1], [0, 1, 1], [1, -1, 1], [1, 0, 1], [1, 1, 1]]
sage: cube1 = Cone(vertices=V)
sage: cube1.Triangulation()
[[[0L, 1L, 3L], 1L],
 [[1L, 2L, 3L], 1L],
 [[2L, 3L, 4L], 1L],
 [[2L, 4L, 5L], 1L],
 [[3L, 4L, 6L], 1L],
 [[4L, 5L, 6L], 1L],
 [[5L, 6L, 7L], 1L],
 [[5L, 7L, 8L], 1L]]

**Comment by videlec** [comment]
@jplab What do you mean inside Sage? which Normaliz version? which PyNormaliz version?

**Comment by jplab** [comment]
I have the latest version of I am on the branch of the ticket 27682, with the latest commit of pynormaliz on github, and normaliz 3.7.0.

I just executed the earliest code in "sage -python" and got exactly the segfault after `cube1.Triangulation()`.

Executing in sage:

`sage: from PyNormaliz import Cone, NmzCone, NmzResult
sage: V = [[-1, -1, 1], [-1, 0, 1], [-1, 1, 1], [0, -1, 1], [0, 0, 1], [0, 1, 1], [1, -1, 1], [1, 0, 1], [1, 1, 1]]
sage: cube2 = NmzCone(vertices=V)
sage: NmzResult(cube2, "SupportHyperplanes") == [[-1, 0, 1], [0, -1, 1], [0, 1, 1], [1, 0, 1]]
True
sage: NmzResult(cube2, "Volume") == [[8, 1]]
True
sage: ------------------------------------------------------------------------
/home/jplabbe/sage/local/lib/python2.7/site-packages/cysignals/signals.so(+0x79e5)[0x7fbb9a4c19e5]
[snip]
------------------------------------------------------------------------
Unhandled SIGSEGV: A segmentation fault occurred.
This probably occurred because a *compiled* module has a bug
in it and is not properly wrapped with sig_on(), sig_off().
Python will now terminate.
------------------------------------------------------------------------
/usr/local/bin/sage: line 3: 26664 Segmentation fault      ~/sage/./sage $@`

**Comment by videlec** [comment]
Fixed by  997ae34
